@xen-orchestra/web/src/components/vm/VmActionItems/VmActionPowerStateItems.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <MenuItem icon="fa:play" :busy="isVmBusyToStart" :disabled="!isVmHalted" @click="start">
+    {{ t('start') }}
+  </MenuItem>
+</template>
+
+<script lang="ts" setup>
+import { useVmStartJob } from '@/composables/vm/xo-vm-start.composable'
+import { isVmOperatingPending } from '@/utils/xo-records/vm.util'
+import MenuItem from '@core/components/menu/MenuItem.vue'
+import { VM_OPERATIONS, VM_POWER_STATE, type XoVm } from '@vates/types'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const props = defineProps<{ vm: XoVm }>()
+
+const { t } = useI18n()
+
+const selectedVms = computed(() => [props.vm])
+
+const isVmHalted = computed(() => props.vm.power_state === VM_POWER_STATE.HALTED)
+
+const { run: runVmStart, canRun: canRunVmStart } = useVmStartJob(selectedVms)
+
+const areOperationsPending = (operation: VM_OPERATIONS | VM_OPERATIONS[]) =>
+  selectedVms.value.some(vm => isVmOperatingPending(vm, operation))
+
+const isVmBusyToStart = computed(() => areOperationsPending(VM_OPERATIONS.START))
+
+const start = () => {
+  if (!canRunVmStart.value || !isVmHalted.value) return
+  void runVmStart()
+}
+</script>
```

Comment:
```suggestion
<template>
  <MenuItem icon="fa:play" :busy="isBusy" :disabled="!canStart" @click="start">
    {{ t('start') }}
  </MenuItem>
</template>

<script lang="ts" setup>
import { useVmStartJob } from '@/jobs/vm/vm-start.job.ts'
import { isVmOperatingPending } from '@/utils/xo-records/vm.util.ts'
import MenuItem from '@core/components/menu/MenuItem.vue'
import { VM_OPERATIONS, VM_POWER_STATE, type XoVm } from '@vates/types'
import { computed } from 'vue'
import { useI18n } from 'vue-i18n'

const { vm } = defineProps<{
  vm: XoVm
}>()

const { t } = useI18n()

const { run: start, canRun, isRunning } = useVmStartJob([vm])

const isVmHalted = computed(() => vm.power_state === VM_POWER_STATE.HALTED)

const isBusy = computed(() => isRunning.value || isVmOperatingPending(vm, VM_OPERATIONS.START))

const canStart = computed(() => isVmHalted.value && canRun.value && !isBusy.value)
</script>
```

I suggest something like that, what do you think ?